darts/timeseries.py

```diff
@@ -3620,6 +3620,7 @@ def plot(
         high_quantile: Optional[float] = 0.95,
         default_formatting: bool = True,
         label: Optional[Union[str, Sequence[str]]] = "",
+        plot_all_components: Optional[bool] = False,
```

Comment:
Could we maybe replace this by `max_nr_components: int = 10`, which would configure the maximum number of components to plot? We can also say that `-1` means all components.

Reply:
I have changed to code to use `max_nr_components` as suggested